src/app/app.module.js

```diff
@@ -1,17 +1,27 @@
 //Angular and Angularmd
 import angular from 'angular';
-import angularmd from 'angular-material';
-import angularanimate from 'angular-animate';
-import angulararia from 'angular-aria';
-import angularmessages from 'angular-messages';
+import angularMd from 'angular-material';
+import angularAnimate from 'angular-animate';
+import angularAria from 'angular-aria';
+import angularMessages from 'angular-messages';
+import 'angular-material/angular-material.scss';
```

Comment:
it's a good practice add and empty line between different imports blocks (I mean , by type, libraries, styles, services, etc), just for readability